workspace/popjava/src/popjava/service/jobmanager/network/POPNetworkNodeFactory.java

```diff
@@ -0,0 +1,25 @@
+package popjava.service.jobmanager.network;
+
+import java.util.List;
+import popjava.service.jobmanager.protocol.POPConnectorJobManager;
+import popjava.service.jobmanager.protocol.POPConnectorDirect;
+import popjava.util.Util;
+
+/**
+ *
+ * @author Davide Mazzoleni
+ */
+public class POPNetworkNodeFactory {
+	public static POPNetworkNode makeNode(List<String> other) {
+		String connector = Util.removeStringFromList(other, "connector=");
+		// use job manager if nothing is specified
+		if (connector == null)

```

Comment:
Add { and } around the if.
This is the code style we try to adhere to (even if some older code has not yet been moved to this style)